Generator_pgn.py

```diff
@@ -0,0 +1,82 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+
+class Generator_pgn():
+    def __init__(self,replay_queue: mp.Queue,pgn_path):
+        
+        r"""
+        Iterable class that generates stochastic fen and value samples by parallel running generators.
+        :param replay_queue: target queue for samples
+        :param current_line: currently used line from the data file, representing one game
+        :param board: the chessboard on which the moves are made
+        :param pgn_paths: list of paths to data files
```

Comment:
:param:  should only describe function (in this case, constructor) parameters. If you want to describe what a variable (in this case, class private variable) does, use a comment instead